cmd/kritis/signer/main.go

```diff
@@ -44,22 +44,28 @@ var (
 	mode               string
 	image              string
 	vulnzTimeout       string
-	priKeyPath         string
-	passphrase         string
 	policyPath         string
 	attestationProject string
 	noteName           string
+	// pgp key flags
+	pgpPriKeyPath string
+	pgpPassphrase string
+	// kms flags
+	kmsKeyName   string
+	kmsDigestAlg string
 )
 
 func init() {
 	flag.StringVar(&mode, "mode", "check-and-sign", "mode of operation, check-and-sign|check-only|bypass-and-sign")
 	flag.StringVar(&image, "image", "", "image url, e.g., gcr.io/foo/bar@sha256:abcd")
 	flag.StringVar(&vulnzTimeout, "vulnz_timeout", "5m", "timeout for polling image vulnerability , e.g., 600s, 5m")
-	flag.StringVar(&priKeyPath, "private_key", "", "signer private key path, e.g., /dev/shm/key.pgp")
-	flag.StringVar(&passphrase, "passphrase", "", "passphrase for private key, if any")
+	flag.StringVar(&pgpPriKeyPath, "pgp_private_key", "", "pgp private signing key path, e.g., /dev/shm/key.pgp")
```

Comment:
Not for this PR, I'm just curious about your opinion.

These flags are getting complicated, do you think it makes sense to have a config file instead?  Maybe using --policy would work, as long as the vulnz policy is clearly separated from key specs etc.

Reply:
Yeah, I think adding the option of config file does make sense. The two (flags and config file) can probably co-exist.